src/main/java/edu/ie3/util/geo/GeoUtils.java

```diff
@@ -98,795 +78,4 @@ protected GeoUtils() {
     Coordinate coordinate = new Coordinate(x, y, 0);
     return DEFAULT_GEOMETRY_FACTORY.createPoint(coordinate);
   }
-
-  /**
-   * This method takes all {@link RelationMember}s and joins the given ways to have closed ways.
-   * This only works, if the ways are connected all in a line.
-   *
-   * @param relation {@link Relation} relation to treat
-   * @return Deep copy of the {@code relation} with closed ways
-   * @deprecated This method is currently not under test and has to be revised thoroughly
```

Comment:
This whole class is not deprecated bc it should be removed but as an indicator, that the methods ARE WORKING but NOT TESTED! Hence, I would keep them because we need them.